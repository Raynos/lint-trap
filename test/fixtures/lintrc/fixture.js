'use strict';

var foo = function() {};

module.exports = foo;
