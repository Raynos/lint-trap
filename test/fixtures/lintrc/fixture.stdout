fixture.js
    warning  eslint     3:10   Missing function expression name.  func-names
